Servers/UI/OJS.Servers.Ui/ClientApp/src/components/contests/contests-filters/ContestFilters.tsx

```diff
@@ -78,6 +92,41 @@ const ContestFilters = ({ onFilterClick }: IContestFiltersProps) => {
                 ? groupFilters
                 : filteredStrategyFilters;
 
+            if (type === FilterType.Strategy) {
+                return (
+                    <div style={{ marginTop: 15 }}>
+                        <InputLabel id="strategy-label">Strategy</InputLabel>
+                        <Select
+                          sx={{
+                              width: 350,
```

Comment:
Is it possible for this to be a scss module?

Reply:
its either impossible or very hard to achieve styiling components from libraries with projects css, even if it works out, a lot of !important rules should be used which by itself is a bad practice, that's why styling components from libraries happens from their properties provided by the library itself